app/scripts/lib/service-name.js

```diff
@@ -13,6 +13,7 @@ define(function (require, exports, module) {
   }
 
   var serviceNames = {
+    amo: t('Mozilla Add-ons'),
```

Comment:
This name I believe should already be set as `relier.serviceName` and filled in when we [get the OAuth relier info](https://github.com/mozilla/fxa-content-server/blob/0c84ed2695abf0411c875aa499f61be38e191464/app/scripts/models/reliers/oauth.js#L166). I don't believe this line is needed.